### PR TITLE
DOC-3462 sql txn contended metric

### PR DESCRIPTION
**Pull request by andf-crl**
Addresses: DOC-3462

- Going to start merging metrics additions directly; they don't appear to interest many reviewers.

### CODE REVIEW

File: _includes/v22.2/metric-names.md [hunk 1]
```diff
@@ -247,6 +247,7 @@ Name | Help
 `sql.txn.abort.count` | Number of SQL transaction ABORT statements
 `sql.txn.begin.count` | Number of SQL transaction BEGIN statements
 `sql.txn.commit.count` | Number of SQL transaction COMMIT statements
+`sql.txn.contended.count` | Number of SQL transactions experienced contention
```
Comment (taroface) [inline _includes/v22.2/metric-names.md:250]: Nit: `Number of SQL transactions that experienced contention`

**Review by taroface — COMMENTED** [review]
I know this is merged, but I have one nit. Happy to review these if no one else wants to.

File: _includes/v22.2/metric-names.md (hunk 1 above)
Comment (andf-crl) [reply _includes/v22.2/metric-names.md:250]: Metrics on this page are (or rather: will shortly be) auto-generated based on `chartcatalog` output. As such, manual efforts are overwritten by verbatim upstream each time I update this file (at each minor rev, or per ticket adding new metric: whichever sooner).

I'll edit this one by hand, as we're not yet fully automated here. But the real (only) fix is to get engineering to revisit these descriptions. [Awaiting upstream response](https://github.com/cockroachdb/cockroach/issues/92925)!

Comment (taroface) [reply _includes/v22.2/metric-names.md:250]: Oh, I see! Thank you!

**Pull request merged**